website_cms/templates/sidebar.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+<template id="sidebar_editable" name="CMS sidebar editable">
+
+<!--   <div id="sidebar_content"
+       class="oe_structure"
+       t-att-data-oe-model="main_object._name"
+       t-att-data-oe-field="'sidebar_content'"
+       t-att-data-oe-id="main_object.id"
+       data-oe-type="html">
+
+    <t t-if="not main_object.sidebar_content">
+      <mark>Customize this block as you like</mark>
+    </t>
+  </div> -->
+
+  <t t-if="False"># Make `sidebar_content` editable ;)</t>
+  <div id="sidebar_content"
+       t-field="main_object.sidebar_content"
+       class="oe_structure"
+       t-att-contenteditable="editable and 'true' or None"
+       />
+
+</template>
+
+<!-- enable view for cms sidebar -->
+<record id="website_cms.sidebar_editable" model="ir.ui.view">
+  <field name="cms_sidebar" eval="1" />
+</record>
+
+
+<template id="sidebar_rendering" name="CMS sidebar rendering">
+
+
+  <t t-if="website.safe_hasattr(main_object, 'sidebar_view_ids')">
+    <t t-if="False">
+      main_object.sidebar_view_ids => throws an error
+      since there's no safe "hasattr" when evaluating
+
+      QWebException: "'NoneType' object is not callable" while evaluating
+      "main_object and getattr(main_object, 'sidebar_view_ids', None)"
+    </t>
+
+    <t t-foreach="main_object.sidebar_view_ids" t-as="view">
+      <t t-call="#{ view.key }" />
```

Comment:
although we might find a way to manage this better on backend (right now the sidebar_view_ids is not visible at all)